.github/actions/build-docker/action.yml

```diff
@@ -26,11 +26,11 @@ inputs:
     description: 'If true, will push the image'
     required: false
     default: 'false'
-  platforms:
+  architectures:
     # See https://docs.docker.com/build/ci/github-actions/examples/#multi-platform-images
-    description: 'Comma separated-list of platforms to build the image for; defaults to linux/amd64'
+    description: 'Comma separated-list of linux architectures to build the image for; defaults to amd64'
     required: false
-    default: 'linux/amd64'
+    default: 'amd64'
```

Comment:
:wrench: If we're only using this input to transform it into `platform`, so we now "save" keystrokes for the `linux` part, I would stay with `linux/amd64`. It doesn't bring much value to omit it, and keeping it remains closer to the expected usage with Docker. I don't think we need to abstract this param. Let me know if I missed further usage though.

EDIT: is it to align it with the `verify` action which expects the parameter to be `amd64` instead of `linux/amd64`?

Reply:
exactly, I moved the arch->platform conversion to the build-docker action to only have it there, in contrast to having to care about arch and platform formats in the calling workflows